server/src/types/grades.ts

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+export interface Grade {
+  attainmentId: number,
+  points: number
+}
+
+export interface Student {
+  id?: number,
+  studentNumber: string,
+  grades: Array<Grade>
+}
+
+export interface AttainmentGrade {
+  userId: number,
+  attainableId: number,
+  points: number
```

Comment:
A "grade" can be either points or, for instance, a final course grade. For the latter "points" is not an appropriate term, so this shouldn't be `points`. Maybe `grade`, unless you have a better idea.

Reply:
I am using `points` as the database model uses `points` instead of `grades`.

Reply:
Then I think the database model should also be updated, although that's not critical for this PR, it can be changed later.

Reply:
Agree, but better to handle this in another PR.